base/errorshow.jl

```diff
@@ -781,7 +781,7 @@ function show_backtrace(io::IO, t::Vector)
 
     try invokelatest(update_stackframes_callback[], filtered) catch end
     # process_backtrace returns a Vector{Tuple{Frame, Int}}
-    frames = (first.(filtered))::Vector{StackFrame}
+    frames = map(x->first(x)::StackFrame, filtered)
```

Comment:
```suggestion
    frames = StackFrame[first(x) for x in filtered]
```

Reply:
Actually these two are different. My version asserts that `first(x)` is already a `StackFrame`; your version will call `convert(StackFrame, x::Any)::StackFrame`. Given that we know `x` is already a `StackFrame`, that makes my version less vulnerable to `convert` invalidations.